llvm/lib/Target/SPIRV/SPIRVBuiltins.h

```diff
@@ -37,6 +37,18 @@ std::optional<bool> lowerBuiltin(const StringRef DemangledCall,
                                  const Register OrigRet, const Type *OrigRetTy,
                                  const SmallVectorImpl<Register> &Args,
                                  SPIRVGlobalRegistry *GR);
+
+/// Translates a string representing a SPIR-V or OpenCL to a TargetExtType that
```

Comment:
Maybe "SPIR-V or OpenCL" -> "SPIR-V or OpenCL builtin type"?

Reply:
Right, did not notice this :)